packages/mobile/src/services/offline-downloader/offline-storage.ts

```diff
@@ -168,12 +167,11 @@ export const getTrackJson = async (
   trackId: string
 ): Promise<Nullable<Track & UserTrackMetadata>> => {
   try {
+    if (!(await verifyTrack(trackId))) return null
```

Comment:
nit, but imo i dont love awaits in if's, so maybe `const isVerified` beforehand?